{
	"schema_version": "1.6.0",
	"id": "UBUNTU-CVE-2014-1938",
	"modified": "2025-01-13T10:21:05Z",
	"published": "2019-11-21T15:15:00Z",
	"related": [
		"CVE-2014-1938"
	],
	"details": "python-rply before 0.7.4 insecurely creates temporary files.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Ubuntu:16.04:LTS",
				"name": "python-rply",
				"purl": "pkg:deb/ubuntu/python-rply@0.7.4-1?arch=source&distro=xenial"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0.7.4-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "pypy-rply",
						"binary_version": "0.7.4-1"
					},
					{
						"binary_name": "python-rply",
						"binary_version": "0.7.4-1"
					},
					{
						"binary_name": "python3-rply",
						"binary_version": "0.7.4-1"
					}
				],
				"ubuntu_priority": "low"
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/cve/2014/UBUNTU-CVE-2014-1938.json"
			}
		}
	],
	"references": [
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2014-1938"
		},
		{
			"type": "REPORT",
			"url": "https://www.cve.org/CVERecord?id=CVE-2014-1938"
		}
	]
}
